MidasCivil_Engine/Compute/MidasMetaData.cs

```diff
@@ -0,0 +1,83 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ComponentModel;
+using BH.oM.Adapters.MidasCivil;
+using BH.oM.Reflection.Attributes;
+
+namespace BH.Engine.Adapters.MidasCivil
+{
+    public static partial class Create
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        [Description("Assigns meta data to a file.")]
+        [Input("location", "Where is the project based")]
+        [Input("description", "Any information about the project, or the script that maybe useful to others.")]
```

Comment:
```suggestion
        [Input("description", "A short description of the project or the script that maybe useful to others.")]
```

Is there a character limit? It is worth mentioning.

Reply:
Updated, no character limit (well, none that anyone will realistically ever hit)

Reply:
Where is the description stored in MidasCivil? Is it the comment box, because that has a 500 character limit - suggest you add a character limit here: https://github.com/BHoM/MidasCivil_Toolkit/blob/1c4696f420df33e1c3967e71e814d8d12e3e3d48/MidasCivil_Adapter/MidasCivilAdapter.cs#L121